cmd/conduit/internal/initialize/init.go

```diff
@@ -106,19 +119,21 @@ func runConduitInit(path string, importerFlag string, processorsFlag []string, e
 
 	config := fmt.Sprintf(sampleConfig, importer, processors, exporter)
 
-	_, _ = f.WriteString(config)
-
+	_, err := configWriter.Write([]byte(config))
 	if err != nil {
 		return fmt.Errorf("runConduitInit(): failed to write sample config: %w", err)
 	}
 
-	fmt.Printf("A data directory has been created %s.\n", location)
-	fmt.Printf("\nBefore it can be used, the config file needs to be updated with\n")
-	fmt.Printf("values for the selected import, export and processor modules. For example,\n")
-	fmt.Printf("if the default algod importer was used, set the address/token and the block-dir\n")
-	fmt.Printf("path where Conduit should write the block files.\n")
-	fmt.Printf("\nOnce the config file is updated, start Conduit with:\n")
-	fmt.Printf("  ./conduit -d %s\n", path)
+	// If a data dir is created, print some additional help.
+	if path != "" {
```

Comment:
nit: it may be better to use `configWriter==nil` check here; one fewer var to track.  `path` is getting updated at line 58 and it only happens when `configWriter == nil`.

Reply:
configWriter is set to the file on line 74. I agree that the logic is strange, maybe I should just refactor to avoid the strange logic.